### ISSUE TITLE
Lucid creation problem or no

### ISSUE THREAD

**Issue by Deoclecian** [Haidra-Org/Lucid-Creations#107]
Img2img problem has dont work times select image?

**Comment by db0** [comment]
Sorry can you rephrase this?

**Comment by bubblelung** [comment]
> Yes, because I switched to a OS-native browser.. The browser can't access the OS anyway so it was never working

sorry I'm sort of a noob... by browser I mean finder window... so it seems to work fine for the basic panel; the 'Load from file' button will open the finder to load a .json from disk..

**Comment by Deoclecian** [comment]
Now works!! lucid creations nothing errors!
